### PR TITLE
added terms to domains, past_projects, and target_markets

### PR DESCRIPTION
**Pull request by omjadas**

### CODE REVIEW

File: target_markets.txt [hunk 1]
```diff
@@ -82,6 +83,7 @@ the elderly
 people who still use hotmail
 people who still use AOL
 people who still pay with checks
+people who still use ISP provided email
```
Comment (qguv) [inline target_markets.txt:94]: YES PERFECT

Comment (omjadas) [reply target_markets.txt:94]: thanks!

**Comment by qguv** [comment]
You're resolving conflicts faster than I can! :smile:

**Review by qguv — APPROVED** [review]

**Comment by qguv** [comment]
Thanks for the additions, @omjadas!

**Pull request merged**